plat/hisilicon/poplar/include/hi3798cv200.h

```diff
@@ -63,11 +63,11 @@
 #define EMMC_CLK_50M			(1 << 8)
 #define EMMC_CLK_25M			(2 << 8)
 
-#define EMMC_DESC_SIZE			(0xF0000)
+#define EMMC_DESC_SIZE			(0x00100000) /* 1MB */
```

Comment:
Static check - Add U / ULL suffix. Could use the U() ULL() macro from utils_def.h